Orchard1/Restart-Site/Restart-Site.psm1

```diff
@@ -43,27 +43,27 @@ function Restart-Site
 		$whiteListFolders = @("\src\Orchard.Azure\Orchard.Azure.CloudService\Orchard.Azure.WebContent\Bin")
 
 		Get-ChildItem -Path ($Path + "\src\") -Recurse |
-		Where-Object { $PSItem.PSIsContainer -and ( $PSItem.Name -eq "bin" -or $PSItem.Name -eq "obj") } |
-		ForEach-Object {
-			if($whiteListFolders.Contains($PSItem.FullName.Substring($Path.Length)))
-			{
-				Get-ChildItem -Path $PSItem.FullName -Recurse -File |
-				ForEach-Object {
-					if(!$whiteList.Contains($PSItem.FullName.Substring($Path.Length)))
+			Where-Object { $PSItem.PSIsContainer -and ( $PSItem.Name -eq "bin" -or $PSItem.Name -eq "obj") } |
```

Comment:
These files have tab indentation instead of spaces, please expand them.
- ./Utility/Orchard1/Restart-Site/Restart-Site.psm1
- ./Readme.md

Reply:
Thanks, good catch! I noticed a lot of incorrect indentations overall and fixed them, but some slipped through.
I just discovered that some of them remain (even though I applied automatic formatting in VS Code), because the `editor.detectIndentation` setting was enabled, so it will try to preserve the (incorrect) indentation, even though the indentation settings are otherwise correct. Updated https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2161770522/Visual+Studio+Code#Recommended-settings according to this.